department-of-veterans-affairs/caseflow
# [WIP] Comment deep links

Resolves #8396

### Description
Adds the ability to access and follow a link to a comment with only the comment's database id.

### Acceptance Criteria
- [x] Add a share button next to the edit and delete buttons which opens a modal with the deep link to that comment.
- [x] Link created opens document containing the annotation
- [x] When navigating to the deep linked comment, the page should scroll to the comment after loading.

### Testing Plan
1. Go to any reader document longer than one page
2. Create a comment on any page other than the first page
3. Click share next to created comment
4. Click the link in the modal to copy it to your clipboard
5. Follow the copied link and confirm the newly created comment is shown.
